src/dlls/mscoree/unixinterface.cpp

```diff
@@ -178,13 +178,13 @@ static void ExtractStartupFlagsAndConvertToUnicode(
 //
 extern "C"
 int coreclr_initialize(
-            const char* exePath,
```

Comment:
I thought the coding convention was 8 space indentation.  At least a majority of the code uses 8 spaces when the function parameters are put on a separate line.

But I'll revert these changes if you want.

mikem

From: Jan Vorlicek<mailto:notifications@github.com>
Sent: ?Monday?, ?December? ?14?, ?2015 ?12?:?49? ?AM
To: dotnet/coreclr<mailto:coreclr@noreply.github.com>
Cc: Michael J McLaughlin<mailto:mikem@microsoft.com>


In src/dlls/mscoree/unixinterface.cpp<https://github.com/dotnet/coreclr/pull/2327#discussion_r47472228>:

> @@ -178,13 +178,13 @@ static void ExtractStartupFlagsAndConvertToUnicode(
>  //
>  extern "C"
>  int coreclr_initialize(
> -            const char* exePath,


These indents were intentional to not to align the parameters with the name of the function / function code lines. Could you please revert this change?

-
Reply to this email directly or view it on GitHub<https://github.com/dotnet/coreclr/pull/2327/files#r47472228>.